### DISCUSSION TITLE
Eliminate append command

### DISCUSSION THREAD

**Discussion by stureandersen** [NilsSve/DfRefactor#46] [Ideas]
append command -> move string expression

**Comment by NilsSve** [comment]
???
Is this really needed?
At one moment, John Tuohy did list "Append" as "obsolete" but later changed his mind because so many developers love this command because it can make the code easier to read.

One example is when concatenating SQL strings together.

**Comment by stureandersen** [comment]
I just find it odd that APPEND is the only string operation you should do with a command rather than a string expression.

Anyway, it can be switched off under "refactoring functions".

**Comment by NilsSve** [comment]
Fair enough. I'll add it to the "New Refactoring Functions ToDo" list.

**Comment by NilsSve** [comment]
Closed.

**Comment by NilsSve** [comment]
How should the change from an Append to an Append function work if the source line looks something like this;
    Append sSource "Some text string" "Another text string" sString2
???
